Topic1_Audio_Player/src/State.java

```diff
@@ -0,0 +1,4 @@
+
+public interface State {
```

Comment:
Warning: Consider refactor the name to something like audioPlayerCommand (Command pattern). State doesn't give the idea of a class running an specific operation (the commands, like play, stop, etc).